### DISCUSSION TITLE
Fetching live liquidity positions

### DISCUSSION THREAD

**Discussion by alexskliu** [uniswap-python/uniswap-python#116] [Q&A] [answered]
Thanks for the awesome work @ErikBjare. Is there a function to query live positional balance of a wallet when providing liquidity? Ie. LP pool provided at the beginning is 2500 USD and 1 ETH but as the price goes down, the position becomes 2000USD and 1.2 ETH etc.

**Comment by ErikBjare** [comment, marked as answer]
There is not, but there's code in the uniswap-v3-sdk that does what you want: https://github.com/Uniswap/uniswap-v3-sdk/blob/25f4128f7afa42cfd5527a9d422e8751e99f4f44/src/entities/position.ts#L65-L127
